### DISCUSSION TITLE
Entrance to register account

### DISCUSSION THREAD

**Discussion by hupo376787** [rustdesk/rustdesk#6558] [General]
Hi, where is the entrance to register
![image](https://github.com/rustdesk/rustdesk/assets/14960168/745cb91e-2a74-4765-a94c-5112c95d1b31)

**Comment by rustdesk** [comment]
https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/

**Comment by Arcanjo-Gabriel** [comment]
Can't register, why make registration so hard? Back to team viewer.
